test/services/notifications/roNotificationHandler.test.js

```diff
@@ -2,12 +2,18 @@ const createRoNotificationHandler = require('../../../server/services/notificati
 const transitionForDestinations = require('../../../server/services/notifications/transitionsForDestinations')
 const { STAFF_NOT_LINKED } = require('../../../server/services/serviceErrors')
 
+const { createPrisonerServiceStub } = require('../../mockServices')
+
 describe('roNotificationHandler', () => {
+  /** @type {any} */
   let roNotificationSender
   let licenceService
   let prisonerService
+  /** @type {any} */
   let eventPublisher
+  /** @type {any} */
```

Comment:
shouldn't have to use any as much if we create shareable mock implementations  for them